NetKAN/ProbeControlRoom.netkan

```diff
@@ -0,0 +1,32 @@
+{
+    "spec_version": "1.2.1.0",
```

Comment:
"spec_version" refers to the CKAN spec version, not the mod version. This should be:

    "spec_version": 1,